src/routes/blogs.rs

```diff
@@ -347,7 +347,15 @@ pub fn outbox(name: String, rockets: PlumeRocket) -> Option<ActivityStream<Order
     let blog = Blog::find_by_fqn(&rockets, &name).ok()?;
     Some(blog.outbox(&*rockets.conn).ok()?)
 }
-
+#[allow(unused_variables)]
+#[get("/~/<name>/outbox?<page>")]
+pub fn outbox_page(
+    name: String,
+    page: Page,
+    _rockets: PlumeRocket,
+) -> Option<ActivityStream<OrderedCollectionPage>> {
+    None
```

Comment:
Could you please call `Blog::outbox_page` here, even if it doesn't do anything yet?

Reply:
Sure thing.